BigChocoMonster/mirai-ui
# Initial setup

#### MONDAY ticket number?
MIR-0002 Set up UI Project
https://mirai--team.monday.com/boards/206182634/pulses/206232711

#### What does this PR do?
Sets up the angular application for the project

#### Where should the reviewer start?
Code changes

#### How should this be manually tested?
- Clone this repo
- Run `npm i` in folder terminal
- Run `ng serve` in folder terminal
- Open `http://localhost:4200/` in browser
_The application should be running (Screenshot attached)_

#### Any background context you want to provide?

#### Screenshots (if appropriate)
![Screenshot 2019-03-27 at 7 40 09 PM](https://user-images.githubusercontent.com/40193403/55082713-2b33b100-50c8-11e9-9240-d2542e00d7ef.png)
